openstf/model/model_creator.py

```diff
@@ -6,21 +6,31 @@
 from lightgbm import LGBMRegressor
 
 from openstf.enums import MLModelType
+from openstf.model.xgb_quantile import XgbQuantile
 
 
 class ModelCreator:
+
     """Factory object for creating model trainer objects"""
 
     # Set object mapping
     MODEL_TRAINER_CONSTRUCTORS = {
         MLModelType.XGB: XGBRegressor,
         MLModelType.LGB: LGBMRegressor,
+        MLModelType.XGB_QUANTILE: XgbQuantile,
     }
 
     @staticmethod
-    def create_model(model_type: MLModelType) -> RegressorMixin:
+    def create_model(pj: dict) -> RegressorMixin:
         # check if model type is valid
-        if model_type not in [k.value for k in ModelCreator.MODEL_TRAINER_CONSTRUCTORS]:
-            raise KeyError(f"Unknown model type: {model_type}")
+        if pj["model"] not in [
+            k.value for k in ModelCreator.MODEL_TRAINER_CONSTRUCTORS
+        ]:
+            raise KeyError(f"Unknown model type: {pj['model']}")
 
-        return ModelCreator.MODEL_TRAINER_CONSTRUCTORS[MLModelType(model_type)]()
+        if MLModelType(pj["model"]) == MLModelType.XGB_QUANTILE:
```

Comment:
Kunnen we nog een list verzinnen om hier niet een uitzondering voor dit specifieke model type te hoeven maken.?
Bijv dat de model_trainer_constructors ook **kwargs als input toestaan, waar ze niet per se iets mee hoeven te doen. 
En dat de return dan zoiets wordt (niet getest):

```
return ModelCreator.MODEL_TRAINER_CONSTRUCTORS[MLModelType(pj["model"])](
                quantiles=tuple(pj.get("quantiles"), None)
                )
```

Reply:
good idea, this just works